brion/plugin/compartmentReportBinary.cpp

```diff
@@ -653,6 +663,10 @@ bool CompartmentReportBinary::_parseMapping()
 
         if (_header.byteswap)
```

Comment:
Maybe, but get if a free function and whether bytes need swapping is a member variable inside the header, so the change you request requires either making get a member of add an argument, and none of the them is a nice change.